<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JEDNOSTKI</class>
 <widget class="QDialog" name="JEDNOSTKI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(14, 8, 73, 255), stop:0.36 rgba(28, 17, 145, 255), stop:0.6 rgba(126, 14, 81, 255), stop:0.75 rgba(234, 11, 11, 255), stop:0.79 rgba(244, 70, 5, 255), stop:0.86 rgba(255, 136, 0, 255), stop:0.935 rgba(239, 236, 55, 255));
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QComboBox" name="druga_BOX"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="pierwsza_line"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="typ_BOX">
     <item>
      <property name="text">
       <string>Wybierz typ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Masa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Długość</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Czas</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="pierwsza_BOX"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QPushButton" name="OBLICZ">
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed{
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>OBLICZ</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QLineEdit" name="display">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
